packages/@ourworldindata/components/src/MarkdownTextWrap/MarkdownTextWrap.tsx

```diff
@@ -15,6 +15,7 @@ import { TextWrap } from "../TextWrap/TextWrap.js"
 import fromMarkdown from "mdast-util-from-markdown"
 import type { Root, Content } from "mdast"
 import { match } from "ts-pattern"
+import { dropRightWhile, dropWhile } from "lodash"
```

Comment:
And this right here will increase bundlesize quite a bit - make sure to import lodash via `Util.ts` always.

We should probably get rid of that annoyance at some point, but for now that's how it goes

Reply:
Really? Our bundlewatch GA didn't show any increase. I thought that importing `{ _ } from "lodash"` was the big no-no. What am I missing?

Reply:
Ah you're right!
I was missing [my own change](https://github.com/owid/owid-grapher/commit/a56e1749ab83d657b66c8da028377eb144cb26a8) half a year ago 😬